wrapper:
  config_file_name: avia.yaml
  lidar_topic: /livox/lidar
  imu_topic: /livox/imu
  lidar_type: 0 # 0)avia
  blind: 0.5
  det_range: 450.0

front_end:
  extrin_r: [1,0,0,
            0,1,0,
            0,0,1]
  extrin_t: [0.4165,0.2326,-0.2840]
  gravity_align: false
  filter_leaf_size: 0.2
  extrinsic_est_en: false
  enable_record: false
  record_file_name: record.txt
ieskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.0001
  cov_bias_gyroscope: 0.0001